families/settings/tests/test_settings.py

```diff
@@ -0,0 +1,118 @@
+import os
+import shutil
+import tempfile
+from collections import OrderedDict
+from sawtooth_settings.processor.config.settings \
+    import merge_settings_config
+from sawtooth_settings.processor.config.settings \
+    import load_toml_settings_config
+from sawtooth_settings.processor.config.settings \
+    import load_default_settings_config
+from sawtooth_sdk.processor.exceptions import LocalConfigurationError
+from sawtooth_processor_test.transaction_processor_test_case \
+    import TransactionProcessorTestCase
+
+
+class TestSettings(TransactionProcessorTestCase):
+
+    def test_load_toml_settings_config(self):
+        """ the test case will create the toml file at a per the environment

```

Comment:
Basic grammar check: Start a sentence with an uppercase letter. There should be no space BEFORE a comma. There should always be a space AFTER a comma. The second line onwards should always be aligned with the starting " of the comment. End the comment with a '.' (period)